packages/console/src/pages/SignInExperience/components/ManageLanguageModal/LanguageEditor.tsx

```diff
@@ -7,14 +7,14 @@ import { useCallback, useContext, useEffect, useMemo } from 'react';
 import { FormProvider, useForm } from 'react-hook-form';
 import { toast } from 'react-hot-toast';
 import { useTranslation } from 'react-i18next';
-import useSWR from 'swr';
+import useSWR, { useSWRConfig } from 'swr';
 
 import Button from '@/components/Button';
 import useApi, { RequestError } from '@/hooks/use-api';
 import Delete from '@/icons/Delete';
+import { CustomPhraseResponse } from '@/types/custom-phrase';
 
-import { CustomPhrasesContext } from '../../hooks/use-custom-phrases-context';
-import { CustomPhraseResponse } from '../../types';
+import { LanguageEditorContext } from '../../hooks/use-language-editor-context';
```

Comment:
This is not supposed to be the way that a hook is used. So considering split the context and provider into separated items.

Reply:
ditto